group_vars/all.yml

```diff
@@ -94,6 +97,9 @@ supported_os:
   Rocky:
     versions:
       - 9.2
+      - 9.1
+      - 9.0
+      - 8.8
```

Comment:
I remember we specifically agreed to not add them since they were untested - and I was for adding them because 99% are working fine